### ISSUE TITLE
Custom Sound Input File url not working? (for urls with space)

### ISSUE THREAD

**Issue by Ocanamat** [RoderickQiu/wnr#125]
Hi,

I am running wnr 1.30.2 on Windows 11 23H2 and I have tried several different URIs to get custom sounds working from my local pc:

- `file:///c/...` and `file:///c:/...`
- `file://c/...` and `file://c:/...`
- `c/...` and `c:/...`
- And windows paths directly: "C:\Program Files\..." 

And none worked. What would be the correct syntax for entering a local file path in this settings?

Kind regards,

**Comment by RoderickQiu** [comment]
C:\Users\Roderick\Downloads\music.mp3 I tested with this and it indeed works. 

> And windows paths directly: "C:\Program Files..."

Maybe you remove the "" mark and try again? It is a problem that my description was not clear enough, though.

**Comment by Ocanamat** [comment]
I had the path without quotes, but the issue is that it doesn't like spaces in the path. I moved the audio files to a C:\ folder that I use for old software that does not like long Windows paths and the timer works as expected.

**Comment by RoderickQiu** [comment]
Yes, this is truly a problem then, that the current system doesn't work with paths with spaces. Thanks for that report.
